src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -388,6 +389,25 @@ trait BeamHelper extends LazyLogging {
     }), scala.concurrent.duration.Duration.Inf)
   }
 
+  def writeScenarioPrivateVehicles(scenario: MutableScenario, beamServices: BeamServices): Unit = {
+    val csvWriter: FileWriter = new FileWriter("householdVehicles.csv", true)
```

Comment:
where is this being written out? root folder?

Reply:
@wrashid yes, it writes to root folder. as per requirement here https://github.com/LBNL-UCB-STI/beam/issues/1571

Reply:
it should go to the output folder, not beam root